test/test-execute/exec-bind-paths.service

```diff
@@ -0,0 +1,14 @@
+[Service]
+Type=oneshot
+# Create a file in /tmp/test-exec_bind_paths
+ExecStart=/bin/sh -c 'touch /tmp/test-exec_bind_paths/thisisasimpletest'
+# Then, the file can be access through /tmp
+ExecStart=/bin/sh -c 'test -f /tmp/thisisasimpletest'
+# Also, through /tmp/test-exec_bind_readonly_paths
+ExecStart=/bin/sh -c 'test -f /tmp/test-exec_bind_readonly_paths/thisisasimpletest'
+# The file cannot modify through /tmp/test-exec_bind_readonly_paths
+ExecStart=/bin/sh -x -c '! /bin/touch /tmp/test-exec_bind_readonly_paths/thisisasimpletest'
```

Comment:
I think it would be better to replace `/bin/touch` with `touch`. I'm sorry for not noticing that earlier.

Reply:
Thanks. Updated.